src/test/utils/EigenLayerUnitTestSetup.sol

```diff
@@ -32,34 +24,15 @@ abstract contract EigenLayerUnitTestSetup is Test, Utils {
     StrategyBase public wethStrat;
     StrategyBase public eigenStrat;
     StrategyBase public baseStrategyImplementation;
-
-    PauserRegistry public pauserRegistry;
-    ProxyAdmin public eigenLayerProxyAdmin;
-    Reenterer public reenterer;
-
-    mapping(address => bool) public addressIsExcludedFromFuzzedInputs;
-
-    address public constant pauser = address(555);
-    address public constant unpauser = address(556);
     uint256 wethInitialSupply = 10e50;
     uint256 public constant eigenTotalSupply = 1e50;
 
-    // Helper Functions/Modifiers
-    modifier filterFuzzedAddressInputs(address fuzzedAddress) {
-        cheats.assume(!addressIsExcludedFromFuzzedInputs[fuzzedAddress]);
-        _;
-    }
-
-    function setUp() public virtual {
+    function setUp() public virtual override {
```

Comment:
This looks good - although I don't think the strategies need to be deployed here as those are just specific to unit tests for strategies and not other files

Reply:
Removed as well !